src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts

```diff
@@ -86,13 +78,11 @@ describe('Exercise Scores Component', () => {
     result.participation = participation;
     result.assessmentType = AssessmentType.MANUAL;
 
-    const router = new MockRouter();
-
     const route = { data: of({ courseId: 1 }), children: [] } as any as ActivatedRoute;
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, RouterTestingModule.withRoutes([]), TranslateTestingModule, FormsModule, NgxDatatableModule],
+            imports: [ArtemisTestModule, RouterTestingModule.withRoutes([]), MockModule(NgxDatatableModule)],
```

Comment:
```suggestion
            imports: [ArtemisTestModule, MockModule(NgxDatatableModule)],
```
Most times this is not needed

Reply:
I tried without but tests were failing. In this case, it seems to be needed.